components/chainhook-cli/Cargo.toml

```diff
@@ -38,9 +38,13 @@ atty = "0.2.14"
 crossbeam-channel = "0.5.8"
 uuid = { version = "1.3.0", features = ["v4", "fast-rng"] }
 threadpool = "1.8.1"
-rocket_okapi = "0.8.0-rc.3"
+rocket_okapi = { version = "0.8.0-rc.3", git = "https://github.com/MicaiahReid/okapi.git", branch = "feat-chainhook-fixes"}
```

Comment:
Could we host the fork under the `hirosystems` org?

Reply:
Why must you thwart my plans for github fame?

Reply:
Fixed!